dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1935,14 +1936,41 @@ public void deleteWorkflow(@ApiParam(hidden = true) @Parameter(hidden = true, na
     @ApiOperation(value = "Handle a release of a repository on GitHub. Will create a workflow/service and version when necessary.", authorizations = {
         @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = Workflow.class, responseContainer = "List")
     public List<Workflow> handleGitHubRelease(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user", in = ParameterIn.HEADER) @Auth User user,
-        @ApiParam(value = "Repository path (ex. dockstore/dockstore-ui2)", required = true) @FormParam("repository") String repository,
-        @ApiParam(value = "Username of user on GitHub who triggered action", required = true) @FormParam("username") String username,
-        @ApiParam(value = "Full git reference for a GitHub branch/tag. Ex. refs/heads/master or refs/tags/v1.0", required = true) @FormParam("gitReference") String gitReference,
-        @ApiParam(value = "GitHub installation ID", required = true) @FormParam("installationId") String installationId) {
+        @Parameter(name = "Repository path (ex. dockstore/dockstore-ui2)", required = true) @FormParam("repository") String repository,
+        @Parameter(name = "Username of user on GitHub who triggered action", required = true) @FormParam("username") String username,
+        @Parameter(name = "Full git reference for a GitHub branch/tag. Ex. refs/heads/master or refs/tags/v1.0", required = true) @FormParam("gitReference") String gitReference,
+        @Parameter(name = "GitHub installation ID", required = true) @FormParam("installationId") String installationId) {
         LOG.info("Branch/tag " + gitReference + " pushed to " + repository + "(" + username + ")");
         return githubWebhookRelease(repository, username, gitReference, installationId);
     }
 
+    @POST
+    @Path("/github/install")
+    @Timed
+    @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
+    @UnitOfWork
+    @RolesAllowed({ "curator", "admin" })
+    @Operation(description = "Handle the installation of our GitHub app onto a repository or organization.", security = @SecurityRequirement(name = OPENAPI_JWT_SECURITY_DEFINITION_NAME), responses = @ApiResponse(responseCode = "418", description = "This code tells AWS Lambda not to retry."))
```

Comment:
Might have lost context, why is there a new endpoint for this as opposed to the existing  release endpoint. 
Is this only for reporting lambda installs?

Reply:
Ya, this is purely for reporting lambda installs. Users could technically see on GitHub which repos they have the app installed on, but thought they might want to know dockstore knows too.